test/fixtures/headings.md

```diff
@@ -10,8 +10,16 @@ Some text
 
 ##### heading 5
 
-## heading with link <a href="https://www.richardianson.com">www.richardianson.com</a>
+# headings with extra styles
+
+## heading with a md link <a href="https://adobe.com">Adobe</a>.
+
+## heading with a HTML link [Adobe](https://adobe.com).
```

Comment:
Should the links in these two headings be switched?

Reply:
Fixed in https://github.com/mhaack/helix-md2jcr/pull/5